apps/music/js/ui/views/player_view.js

```diff
@@ -274,6 +274,7 @@ var PlayerView = {
 
   setRepeat: function pv_setRepeat(value) {
     var repeatClasses = ['repeat-off', 'repeat-list', 'repeat-song'];
+    var repeatLabels = ['repeat-off', 'repeat-list', 'repeat-song'];
```

Comment:
I'd just change the variable name to something like `repeatModes`, and maybe add a comment that they're used for class names and L10N IDs.